frontend/public/components/utils/resource-link.jsx

```diff
@@ -43,17 +48,18 @@ export const resourcePath = (kind, name, namespace = undefined) => {
 
 export const resourceObjPath = (obj, kind) => resourcePath(kind, _.get(obj, 'metadata.name'), _.get(obj, 'metadata.namespace'));
 
-/** @type {React.SFC<{kind: K8sResourceKindReference, name: string, namespace?: string, title: string, displayName?: string, linkTo?: boolean}>} */
-export const ResourceLink = ({kind, name, namespace, title, displayName, linkTo=true}) => {
-  const path = resourcePath(kind, name, namespace);
-  const value = displayName ? displayName : name;
+export const ResourceLink = connectToModel(
```

Comment:
Have to `connect` this now to re-render when API discovery completes, otherwise we don't have the models defined to make a URL.